src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,90 @@
+import { useSearchParams } from 'react-router-dom';
+import { useEffect, useMemo, useState } from 'react';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { getPeopleWithParent } from '../utils/getPeopleWithParent';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const [searchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const preparedPeople = () => {
+    setIsLoading(true);
+    setError(false);
+
+    getPeople()
+      .then((fetchedPeople) => {
+        setPeople(getPeopleWithParent(fetchedPeople));
+      })
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  };
+
+  const visiblePeople = useMemo(() => getFilteredPeople({
+    people,
+    sex,
+    query,
+    centuries,
+    sort,
+    order,
+  }), [people, searchParams]);
+
+  useEffect(() => {
+    preparedPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          {!isLoading && !error && (
+            <div className="column is-7-tablet is-narrow-desktop">
+              <PeopleFilters />
+            </div>
+          )}
+
+          <div className="column">
+            <div className="box table-container">
+              {isLoading && <Loader />}
+
+              {error && (
+                <p data-cy="peopleLoadingError">Something went wrong</p>
+              )}
+
+              {!isLoading && !people.length && !error && (
+                <p data-cy="noPeopleMessage">
+                  There are no people on the server
+                </p>
+              )}
+
+              {!visiblePeople.length && !isLoading && (
+                <p>
+                  There are no people matching the current search criteria
+                </p>
+              )}
+
+              {!isLoading && visiblePeople.length && !error && (
+                <PeopleTable people={visiblePeople} />
+              )}
```

Comment:
change your condition so that when the array is empty then zero is not displayed
![image](https://github.com/mate-academy/react_people-table-advanced/assets/54936044/b6a8e0da-81df-4018-85a6-2bc24141b8ba)